src/ce/crt0.src

```diff
@@ -107,6 +107,47 @@ launcher:
 .webaddress:
 	db	"http://tiny.cc/clibs",0
 
+
+	section .start
+
+mpTmrRange    := 0F20000h
+
+tmr2Counter   := 10h
```

Comment:
Timer 2 is probably the best timer for this, right? I imagine that most programs with custom timer usage would use timer 1, and USB (including usbdrvce?) uses timer 3.

Reply:
That sounds good to me